### PR TITLE
fix: topwallet refresh rate

### PR DESCRIPTION
**Pull request by sirpy**

### CODE REVIEW

**Comment by johnsmith-gooddollar** [comment]
@sirpy @L03TJ3 please look my comment

File: packages/sdk-v2/src/sdk/faucet/react.ts [hunk 1]
```diff
@@ -21,7 +23,8 @@ export const useFaucet = async (refresh: QueryParams["refresh"] = 12) => {
     lastNotification.current = latest?.submittedAt;
```
[the remark this answers is not in the document]
Comment (L03TJ3) [reply packages/sdk-v2/src/sdk/faucet/react.ts:23]: shall we do this after launch?

[the remark this answers is not in the document]
Comment (L03TJ3) [reply packages/sdk-v2/src/sdk/faucet/react.ts:23]: for the launch all we need is the faucet to work as intended

[the remark this answers is not in the document]
Comment (sirpy) [reply packages/sdk-v2/src/sdk/faucet/react.ts:23]: We don't need this as an effect. Balance changes is all we need

**Pull request merged**